crypto/rand/randfile.c

```diff
@@ -280,8 +280,16 @@ const char *RAND_file_name(char *buf, size_t size)
     struct stat sb;
 #endif
 
+    /* Find the directory where the state file should go. */
+#ifdef OPENSSL_SYS_WINDOWS
+    if ((s = getenv("RANDFILE")) == NULL) {
+        s = getenv("TEMP");
+        if (s == NULL)
+            s = getenv("APPDATA");
```

Comment:
That link seems to confirm that data placed in that folder can roam with you: "...the data that applications had placed in AppData\Roaming would follow you to the machine you logged in on...". Doesn't seem like a good idea for this file.